agent/runtime-attach/gradle.lockfile

```diff
@@ -8,7 +8,7 @@ io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.18.0-
 io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.18.0=runtimeClasspath
 io.opentelemetry:opentelemetry-bom-alpha:1.18.0-alpha=runtimeClasspath
 io.opentelemetry:opentelemetry-bom:1.18.0=runtimeClasspath
-net.bytebuddy:byte-buddy-agent:1.12.10=runtimeClasspath
+net.bytebuddy:byte-buddy-agent:1.11.18=runtimeClasspath
```

Comment:
is this correct? should this line be gone too?

Reply:
this is correct, this dependency (and version) is coming from the upstream artifact: `io.opentelemetry.contrib:opentelemetry-runtime-attach-core:1.18.0-alpha`